src/platforms/dotnet/diagnostic-logger.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Troubleshooting and diagnosing issues
+description: "How to enable logging in Sentry SDK for troubleshooting purposes."
+sidebar_order: 20
+---
+
+Sentry has its own logger that it uses to report diagnostics which may be useful when troubleshooting. To enable logging, set the `Debug` option to `true`:
+
+```csharp
+SentrySdk.Init(o =>
+{
+    o.Dsn = "___PUBLIC_DSN___";
+
+    // Enable debug mode to write diagnostic messages
+    o.Debug = true;
+});
```

Comment:
What about skipping the SentrySdk. since that differs on ASP.NET Core, Xamarin etc.
```suggestion
options =>
{
    options.Dsn = "___PUBLIC_DSN___";

    // Enable debug mode to write diagnostic messages
    options.Debug = true;
});
```

Reply:
I would personally find that confusing, I think. I would that the snippets should probably be compilable.

Reply:
You'd need to add an include of sorts and override. But it's hard to maintain.

If anyone went through setting up for their platform (i.e xamarin, aspnet) they know where to add the options.
I agree it's not ideal but being pragmatic, this is simpler. Right now we have many places with the wrong code snippet because assumes bare bones .NET

Reply:
Otherwise please add the code snippet like: `dotnet.mdx` then `dotnet.xamarin.mdx` and `dotnet.aspnetcore.mdx`